src/site/content/en/metrics/inp/index.md

```diff
@@ -54,7 +54,7 @@ INP is a metric that assesses a page's overall responsiveness to user interactio
 A note on how INP is calculated
 {% endDetailsSummary %}
 
-As stated above, INP is calculated by observing all the interactions made with a page. The chosen value is then a [percentile](https://en.wikipedia.org/wiki/Percentile) of those interactions. A formula is then used to choose a high percentile value of those interactions. For pages with few interactions, the interaction with the worst latency (the 100th percentile) is chosen. For pages with many interactions, the 99th or 98th percentile is chosen.
+As stated above, INP is calculated by observing all the interactions made with a page. The interaction with the worst latency is reported as INP, however, one worst interaction is ignored for every 50 interactions. The vast majority of page experiences do not have over 50 interactions.
```

Comment:
I still think we should explain this more. WDYT about this:

```suggestion
As stated above, INP is calculated by observing all the interactions made with a page. For most sites the interaction with the worst latency is reported as INP. However, for pages with large numbers of interactions, random hiccups can result in an unusually high interaction on an otherwise responsive site. The more interactions, the more likely this is to happen. To counter this, and give a better measure of the actual responsiveness for those types of pages, we ignore one highest interaction for every 50 interactions. The vast majority of page experiences do not have over 50 interactions so will report the worst interaction. The 75th percentile of all the page views is then reported as usual, which further removes outliers to give a value that the vast majority of users experience or better.
```